articles/hello-world-the-first-signs-of-practical-io.md

```diff
@@ -0,0 +1,200 @@
+---
+authors: ["Sora Suegami", "Enrico Bottazzi", "Pia Park"]
+title: "Hello, World: the first signs of practical iO"
+image: "cover.webp"
```

Comment:
```suggestion
image: "/articles/hello-world-the-first-signs-of-practical-io/cover.webp"
```

@psedesign this should have full URL like `/articles/hello-world-the-first-signs-of-practical-io/cover.webp`